usefulUtilities/portal/portal.js

```diff
@@ -0,0 +1,40 @@
+//
+//  portal.js
+//  Created by Zach Fox on 2019-05-23
+//  Copyright 2019 High Fidelity, Inc.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function () {    
+    var Portal = function() {};
+
+
+    Portal.prototype = {
+        enterEntity: function (id) {
+            var properties = Entities.getEntityProperties(id, ["userData"]);
+            var userData;
+
+            try {
+                userData = JSON.parse(properties.userData);
+            } catch (e) {
+                console.error("Error parsing userData: ", e);
+            }
+
+            if (userData) {
+                if (userData.destination) {

```

Comment:
Do you need to do a check here to make sure it's a valid destination?  Not sure what happens if you put something wonky in.

Reply:
Nope, we have no way to validate that. We're depending on the script user to supply a valid destination.